app/src/main/java/com/example/moviedb/ui/base/BaseFragment.kt

```diff
@@ -30,4 +46,22 @@ abstract class BaseFragment<T : ViewDataBinding> : Fragment() {
         activity ?: return
         Toast.makeText(activity, message, Toast.LENGTH_LONG).show()
     }
+
+
+    fun showLoadingDialog() {
+        loadingDialog?.let {
+            activity?.apply {
```

Comment:
không nên lồng 2 cái như này =)), hoặc là check if( != null ) 
```suggestion
            fun Activity.showLoadingDialog() {
                 loadingDialog?.let {
```